### DISCUSSION TITLE
IOS build with xcode 15.2

### DISCUSSION THREAD

**Discussion by jbarfell** [MonoGame/MonoGame#8166] [iOS / iPadOS]
Any suggestions on how to get the monogame IOS app to compile after updating to xcode 15.2 on sonoma. I had to update to support IOS 17 and now VS gives me an error about 15.2 not being compatible. Using VS 2022 v17.8.6.

**Comment by ThomasFOG** [comment]
Microsoft is deprecating "VS for Mac". The encouraged route is to use VS Code now. We haven't documented this new route yet, but you'll need the ios workload installed (```dotnet workload install ios```) and [both the "C#" and "C# Dev Kit" extensions for VS Code](https://code.visualstudio.com/docs/languages/dotnet).
